jobs/gtfs-rt-parser-v2/gtfs_rt_parser.py

```diff
@@ -619,6 +631,7 @@ def main(
     threads: int = 4,
     jar_path: Path = JAR_DEFAULT,
     verbose: bool = False,
+    url: str = None,
```

Comment:
is this just for testing, so you can just do one URL? given that below it seems to be a base64 encoded url I would maybe change the argument name?